src/sentry/incidents/models.py

```diff
@@ -401,6 +405,21 @@ def created_by(self):
             pass
         return None
 
+    def owner_id(self):
+        if self.user:
```

Comment:
Should we do something like this?
```
owner = self.user if self.user else self.team
return Actor(owner.id, type(owner)).get_actor_id()
```

Actually, since this is just used by `owner` can we just remove this method entirely and use the method I mention to get the actor?

Reply:
Yeah that sounds better, then we're also not replicating the actor string format here.
Removing owner_id and changing owner to something like:
```
@property
    def owner(self):
        owner = self.user if self.user else self.team
        if owner:
            owner_id = Actor(owner.id, type(owner)).get_actor_id()
            return Actor.from_actor_identifier(owner_id)

        return None
 ```